src/Components/Search/NewSearch/NewSearchInputPillsContainer.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Box,
+  Flex,
+  Pill,
+  ShelfNavigationProps,
+  ShelfNext,
+  ShelfPrevious,
+} from "@artsy/palette"
+import { useRef, useState } from "react"
+import styled, { css } from "styled-components"
+
+const PILLS_OPTIONS = [
+  "TOP",
+  "ARTWORK",
+  "ARTIST",
+  "ARTICLE",
+  "SALE",
+  "ARTIST_SERIES",
+  "COLLECTION",
+  "FAIR",
+  "SHOW",
+  "GALLERY",
+]
+
+const ChevronStyle = css`
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  @media (hover: none) {
+    display: none;
+  }
+`
+
+const PreviousChevron = styled(ShelfPrevious)<ShelfNavigationProps>`
+  ${ChevronStyle}
+`
+
+const NextChevron = styled(ShelfNext)<ShelfNavigationProps>`
+  ${ChevronStyle}
+`
+
+interface GradientBgProps {
+  placement: "right" | "left"
+}
+
+const GradientBg = styled(Box)<GradientBgProps>`
+  position: "absolute";
+  right: 0;
+  width: 100px;
+  height: 30px;
+  background: transparent;
+  background-image: linear-gradient(
+    to ${props => props.placement},
+    rgba(255, 255, 255, 0) 0%,
+    rgba(255, 255, 255, 1) 40%
+  );
+`
+
+const PillsContainer = styled(Flex)`
+  flex-wrap: nowrap;
+  overflow-x: auto;
+  scrollbar-width: none;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+`
+
+export const NewSearchInputPillsContainer = () => {
+  const pillsRef = useRef<HTMLDivElement>(null)
+  const [showPreviousChevron, setShowPreviousChevron] = useState<boolean>(false)
+  const [showNextChevron, setShowNextChevron] = useState<boolean>(true)
+
+  const scrollToLeft = () => {
+    if (pillsRef.current) {
+      pillsRef.current.scrollBy({
+        left: -100,
+        behavior: "smooth",
+      })
+    }
+  }
+
+  const scrollToRight = () => {
+    if (pillsRef.current) {
+      pillsRef.current.scrollBy({
+        left: 100,
+        behavior: "smooth",
+      })
+    }
+  }
+
+  const handleScroll = () => {
+    if (pillsRef.current) {
+      const currentPosition = pillsRef.current.scrollLeft
+      const maxScroll =
+        pillsRef.current.scrollWidth - pillsRef.current.clientWidth
```

Comment:
please also remember that the user can resize the browser window and you have to recalculate this